bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -81,7 +81,7 @@
                 <li><a class="more" href="{{ url('firefox.family.index') }}">{{_('Discover all the ways to take Firefox on the go')}}</a></li>
               </ul>
             </div>
-            <button id="other-platforms-modal-link" class="hidden" type="button">{{_('Download Firefox for another platform')}}</button>
+            <button id="other-platforms-modal-link" class="hidden" type="button">{{_('Firefox for Other Platforms & Languages')}}</button>
```

Comment:
I just realized I still need to include the link to /all for this template, which I totally forgot to do